go/templates/index.tmpl

```diff
@@ -234,38 +234,20 @@
     if (products.includes('assets')) {
       $('#assets').show();
     }
-
-    var linkHandlerCommonOptions = {
-      apiVersion: 'v2',
-      clientName: 'Plaid Quickstart',
-      env: '{{ .plaid_environment }}',
-      product: products,
-      key: '{{ .plaid_public_key }}',
-      countryCodes: '{{ .plaid_country_codes }}'.split(','),
-    };
-    var oauthRedirectUri = '{{ .plaid_oauth_redirect_uri }}';
-    if (oauthRedirectUri != '') {
-      linkHandlerCommonOptions.oauthRedirectUri = oauthRedirectUri;
-    }
-    var oauthNonce = '{{ .plaid_oauth_nonce }}';
-    if (oauthNonce != '') {
-      linkHandlerCommonOptions.oauthNonce = oauthNonce;
-    }
     // This functionality is only relevant for the UK Payment Initiation product.
     if (products.includes('payment_initiation')) {
       $('.payment_initiation').show();
-      $.post('/set_payment_token', {}, function(data) {
+      $.post('/create_link_token_for_payment', {}, function(data) {
         console.log(data);
         var paymentToken = data.payment_token;
         $('#payment_token').text(paymentToken);
```

Comment:
double-check: should we remove payment token from the UI now that link token will be used?

Reply:
the existing code shows the payment token (purely for UI). I preserved that functionality intentionally

Reply:
didn't see `/create_link_token_for_payment` return a payment token tho so my assumption was that `data.payment_token` will return null, perhaps i'm missing something?

Reply:
Yep, we should be removing payment token from the UI as that should no longer be a client-facing concept.
cc @davidzhanghp